### PR TITLE
chore: add MODULE.bazel.lock to .gitignore

### PR DESCRIPTION
**Pull request by 4kangjc**

### CODE REVIEW

**Comment by drigz** [comment]
How have you tested this?

I think https://github.com/google/glog/pull/1116 might be the better fix here - but if you've tested with wasm then this is the right place to change it.

**Comment by 4kangjc** [comment]
> How have you tested this?
> 
> I think #1116 might be the better fix here - but if you've tested with wasm then this is the right place to change it.

I think `DHAVE_UNWIND` and `DHAVE_SYMBOLIZE` are better together, they are stacktrace related macros, maybe it would be better to move them to linux_or_darwin_copts.

**Comment by 4kangjc** [comment]
> How have you tested this?
> 
> I think #1116 might be the better fix here - but if you've tested with wasm then this is the right place to change it.

I just saw #1116. #1116 can indeed solve #1111

**Comment by drigz** [comment]
> I think DHAVE_UNWIND and DHAVE_SYMBOLIZE are better together, they are stacktrace related macros, maybe it would be better to move them to linux_or_darwin_copts.

I have no wasm dev environment and no one has contributed a CI test, so I'd lean towards only changing the options we can test.

**Comment by 4kangjc** [comment]
> > I think DHAVE_UNWIND and DHAVE_SYMBOLIZE are better together, they are stacktrace related macros, maybe it would be better to move them to linux_or_darwin_copts.
> 
> I have no wasm dev environment and no one has contributed a CI test, so I'd lean towards only changing the options we can test.

OK. This PR is changed to only add MODULE.bazel.lock to .gitignore

**Review by drigz — APPROVED** [review]
Thank you!

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 /build*/
 /site/
 bazel-*
+# Bzlmod lockfile
+MODULE.bazel.lock
```
Comment (sergiud) [inline .gitignore:6]: I would guess that the file can only be present in the project root only.
```suggestion
/MODULE.bazel.lock
```

**Review by sergiud — CHANGES_REQUESTED** [review]
Thanks! I have a minor remark though.

File: .gitignore (hunk 1 above)
Comment (4kangjc) [reply .gitignore]: done

**Review by sergiud — APPROVED** [review]
Thanks!

**Pull request merged**